Dockerfile

```diff
@@ -1,19 +1,19 @@
-FROM fedora:29
+FROM fedora:30
```

Comment:
Uhm, but Fedora 30 hasn't been released yet? https://fedoraproject.org/wiki/Releases/30/Schedule

Reply:
Huh - https://hub.docker.com/_/fedora lists rawhide as 31, but latest as 29. That's weird. I'll check it out.

Reply:
https://github.com/fedora-cloud/docker-brew-fedora/tree/c18591956645cd49cd61c50b5f2309b20ff90f52/x86_64

looks like its from here so  still beta

Reply:
Legit. Is that a problem for us?

Reply:
I am fine with that

Reply:
I guess it's fine. Especially if it doesn't corrupt tenant disks all the time :trollface: